### PR TITLE
Enabling tests for python 3, fixing all python 3 incompatibilities.

### LINKED ISSUE #71: Run tests under both python 2 and 3
The CI tests should run under both python 2 and 3

### PR DESCRIPTION
**Pull request by jonparrott**
Fixes #71

### CODE REVIEW

**Comment by jonparrott** [comment]
Lint's failing, will fix.

File: tests/__init__.py [hunk 1]
```diff
@@ -1,132 +1,19 @@
...
-    """Capture stdout."""
-    fake_stdout = StringIO.StringIO()
-    old_stdout = sys.stdout
-
-    try:
-        sys.stdout = fake_stdout
-        yield fake_stdout
-    finally:
-        sys.stdout = old_stdout
+from .utils import (
```
Comment (SurferJeffAtGoogle) [inline tests/__init__.py]: Copyright notice?

File: tests/__init__.py [hunk 2]
```diff
@@ -1,132 +1,19 @@
...
+    BUCKET_NAME_ENV,
```
Comment (SurferJeffAtGoogle) [inline tests/__init__.py]: Can you explain what this code is doing?

File: tests/__init__.py (hunk 1 above)
Comment (jonparrott) [inline tests/__init__.py]: Done.

File: tests/__init__.py (hunk 2 above)
Comment (jonparrott) [inline tests/__init__.py]: I moved all of the code in ``__init__.py`` to ``utils.py``, and added import aliases to the items in ``uitls.py``. This is standard python stuff to find in ``__init__.py``, and doesn't warrant further explanation.

**Comment by jonparrott** [comment]
Lint is fixed, build should pass.

**Comment by jonparrott** [comment]
Build is green, waiting for your word @SurferJeffAtGoogle.

**Comment by SurferJeffAtGoogle** [comment]
LGTM.

**Pull request merged**